pkg/session_recording/recordings.jsx

```diff
@@ -398,6 +450,20 @@
             this.journalctl = null;
         }
 
+        handleDateSinceChange(date) {
+            this.setState({recordingList: []});
+            this.setState({dateSince: date });
+            this.recordingMap = {};
+            this.journalctlStart();
```

Comment:
You need to stop journalctl first, if it's running. Maybe add journalctlRestart for that.